things/calculator/README.md

```diff
@@ -0,0 +1,14 @@
+# Calculator Thing
+
+A simple calculator implementation for the Eclipse Thingweb Test Things project.
+
+## Description
+
+This calculator thing provides basic arithmetic operations through various protocols including HTTP, CoAP, and MQTT.
+
+## Features
+
+-   Basic arithmetic operations (add, subtract, multiply, divide)
+-   Multiple protocol support (HTTP, CoAP, MQTT)
+-   Content negotiation support
+-   Flask-based HTTP implementation
```

Comment:
All readmes should have the same structure. This one misses for example "Protocol Support" and "Usage"